DNN Platform/Library/Framework/CDefault.cs

```diff
@@ -121,7 +121,7 @@ protected string AdvancedSettingsPageUrl
                 if (modules.Count > 0)
                 {
                     var pmb = new PortalModuleBase();
-                    result = pmb.EditUrl(tab.TabID, "", false, "mid=" + modules.ElementAt(0).ModuleID, "popUp=true", "ReturnUrl=" + Server.UrlEncode(TestableGlobals.Instance.NavigateURL()));
+                    result = pmb.EditUrl(tab.TabID, "", false, string.Concat("mid=", modules.ElementAt(0).ModuleID), "popUp=true", string.Concat("ReturnUrl=", Server.UrlEncode(TestableGlobals.Instance.NavigateURL())));

```

Comment:
@bdukes: Yes, you are right. I thought, that there is a difference, but that was a false conclusion. Maybe there a minimal differences between string.empty and "", but I'm not sure if this has any impact on the total performance of the system.

@KenGrierson: Since this pull request changes some base classes there is no explicit UI test. Everything should work as before :)